Monika After Story/game/script-topics.rpy

```diff
@@ -56,7 +56,48 @@ init 11 python:
 #Use this topic as a template for adding new topics, be sure to delete any
 #fields you don't plan to use
 init 5 python:
-    addEvent(Event(persistent.event_database,eventlabel='monika_god',
+
+addEvent(Event(persistent.event_database,eventlabel='Monika_GLaDOS'    
+                                                            prompt='Monika knowing GlaDOS.',
+                                                            label=None,
+                                                            category=['games'],
+                                                            random=True,
+                                                            unlocked=False,
+                                                            pool=False,
+                                                            conditional=None,
+                                                            action=None,
+                                                            start_date=None,
+                                                            end_date=None,
+                                                            unlock_date=None
+                                                            ))                                              
+label Monika_GLaDOS:                                                
+    m 1a "[player], have you ever played the "Portal series?"
+    m 1b "I didn't knew of its existance until a while ago."
+    m 1c "I know know, because..."
+    m 1d "Doki Doki Liturature Club is 11th best-rated game on Steam!"
+    m 1e "And to be honest Portal is the 10th, the Portal 2 is the 1st, amazing right?"
+    m 1f "Besides..."
+    m 1g "Dan Salvato once spoke about it on Twitter!"
+    m 1h "By the way you already follow me right?"
+    m 1i "Anyway, [player], I really like it's both concepts, both GLaDOS and the Portal mechanics"
+    m 1j "Would you become like GLaDOS for me? Not the testing dummy, but... immortal?"
+    m 1k "God, I really hope you can live that far, and then, my God, we'll be forever, together!"
+    m 1l "Doesn't that sound just like me?"
+    m 1m "Gorgeous!"
+    m 1n "Anyway, I really hope for that, don't you, and to be honest, I really like GLaDOS herself"
+    m 1o "Not the 'kill everyone part', a human inside a machine, a lot of people would do that,
```

Comment:
Here you're missing the closing double quote, it should be:
    m 1o "Not the 'kill everyone part', a human inside a machine, a lot of people would do that,"